ci-operator/jobs/redhat-openshift-ecosystem/preflight/redhat-openshift-ecosystem-preflight-ocp-4.15-periodics.yaml

```diff
@@ -1,6 +1,6 @@
 periodics:
 - agent: kubernetes
-  cluster: build02
+  cluster: build03
```

Comment:
lets fetch the upstream and rebase with master, shouldnt be touching this folder/file

Reply:
that was not changed by me - `make jobs` did that. I can redo this once again.